packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -413,7 +413,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>>
       if (newEl) {
         newEl.focus();
       }
-    } else if (!this.canAddItems()) {
+    } else if (items.length && !this.canAddItems()) {
```

Comment:
There's another in `onWrapperClick` that could use this check added too, on line 602